lib/assets/javascripts/utils/popoverHelper.js

```diff
@@ -0,0 +1,8 @@
+import 'bootstrap-sass/assets/javascripts/bootstrap/popover';
+
+$(() => {
```

Comment:
options for the popover can be passed directly in the html link too (e.g. data-animation="fade" data-placement="right").